src/codeenginestudio/thuy/Hex.java

```diff
@@ -0,0 +1,54 @@
+package codeenginestudio.thuy;
+
+public class Hex extends Numberal {
+
+    public Hex(int number) {
+        this.setNumber(number);
+    }
+
+    @Override
+    public boolean equals(Object ob) {
+        if (this == ob) {
+            return true;
+        }
+        if (ob == null || getClass() != ob.getClass()) {
+            return false;
+        }
+        Hex hex = (Hex) ob;
+
+        return getNumber() == hex.getNumber();
+    }
+
+    @Override
+    public Hex minus(int i) {
+        return new Hex(getNumber() - i);
+    }
+
+    public Hex minus(Hex hex) {
+        return new Hex(getNumber() - hex.getNumber());
+    }
+
+    public static int parse(String ob) {
+        String replaceString = "";
+        if (ob.contains(Contants.START_LETTER)) {
+            replaceString = ob.replace(Contants.START_LETTER, Contants.DEFAULT_STRING);
+        }
+
+        return Integer.parseInt(replaceString, Contants.HEX_NUMBER);
```

Comment:
I think we should remove `return` command in here. Do you think so ?

Reply:
I dont think so.. I think return command its ok.

Can you give me more detail?